### ISSUE TITLE
Landscape mode

### ISSUE THREAD

**Issue by ZorkaTheDog** [klimeryk/recalendar.js#70]
Since I got the keyboard folio, I use the remarkable a lot in landscape mode. Is it possible to add lanscape mode as a choice? Or even better merge the two with a link to each other (not sure this is doable...) ?

**Comment by klimeryk** [comment]
Thanks for raising this issue, I have not thought about using it with keyboard folio (since I don't own one), but that totally makes sense.

> Or even better merge the two with a link to each other (not sure this is doable...) ?

Could you explain in more detail what you have in mind here - I'm not sure I understand 🙏 Since this is ultimately just a static PDF, while you could have both versions of the calendar in the same file (horizontal and vertical), the horizontal page for, let's say, April 30 would simply be a completely different page from vertical April 30. There's no way to sync what is written on the horizontal one with the vertical one - since from the perspective of ReMarkable they're just different pages. If I'm understanding what you're asking for correctly, this would only be possible if ReCalendar would be some kind of standalone app on ReMarkable - but that's a whole another level of complexity and would come with lots of drawbacks, I'm afraid.

**Comment by ZorkaTheDog** [comment]
You are right. I was thinking about 2 pages, one for portrait and one for landscape. Didn't think any further at that time.
But still, a landscape version would be nice.

**Comment by klimeryk** [comment]
Just leaving a note here that landscape mode is on my roadmap, but I first need to tackle some bigger changes that will unlock more customizable and easier layout options 🙇
